login-workflow/example/src/screens/AppRouter.tsx

```diff
@@ -94,25 +92,14 @@ export const AppRouter: React.FC = () => {
             <Route
                 path={'/homepage'}
                 element={
-                    <ExperimentalAuthGuard
-                        isAuthenticated={isAuthenticated}
-                        fallbackComponent={<Navigate to={`/login`} />}
-                    >
+                    <ReactAuthGuard isAuthenticated={isAuthenticated} fallBackUrl={'/login'}>
                         <ExampleHome />
-                    </ExperimentalAuthGuard>
-                }
-            />
-            <Route
-                path={'*'}
-                element={
-                    <ExperimentalAuthGuard
-                        isAuthenticated={isAuthenticated}
-                        fallbackComponent={<Navigate to={`/login`} />}
-                    >
-                        <Navigate to={'/login'} />
-                    </ExperimentalAuthGuard>
+                    </ReactAuthGuard>
                 }
             />
+            <Route element={<ReactAuthGuard isAuthenticated={isAuthenticated} fallBackUrl={'/login'} />}>
```

Comment:
@joebochill , need to pass fallbackURL since these screens are out of AuthContextPropvide so routeConfig will not be available.